### ISSUE TITLE
Extension crash in non english Gmail accounts and show nothing

### ISSUE THREAD

**Issue by gisdatasrl** [kevinwucodes/gmail-quick-links#11]
Related to issue: #8 
When I install the extension and enter in gmail.com the extension do this:

```javascript
//TODO: what is the person isn't signed in?  Does this crash extension?
    const currentAccountName = document
      .querySelectorAll('a[title^="Google Account"]')[0]
      .title
      .match(/\(([^)]+)\)/)[1]
```
which causes:
```
Uncaught TypeError: Cannot read property 'title' of undefined
    at index.js:29
```
Unfortunately, I'm italian and such element does not exists because the translation is "Account Google" but if I change the query selector to 
`'a[title^="Account"]'`
It works.

However I suggest you to change the query selector to something more reliable like:
`document.querySelectorAll('a[href*="accounts.google.com"]')`

**Comment by alescdb** [comment]
Same problem here (in french title="Compte Google").
I've tried @gisdatasrl solution and it works !

**Comment by kevinwucodes** [comment]
Wow, this is nice.  I can do this or if @gisdatasrl wants to submit a pull request, I can do that too.

Otherwise, I hope to find some time this weekend to do this.

**Comment by kevinwucodes** [comment]
Fixed in #13
